AUTHORS

```diff
@@ -452,6 +458,7 @@ CONTRIBUTORS:
     Mike Swanson <mikeonthecomputer@gmail.com>
     Milan Jurik <milan.jurik@xylab.cz>
     Minsoo Choo <minsoochoo0122@proton.me>
+    mnrx <mnrx@users.noreply.github.com>
```

Comment:
will be much more better to put real name and email

Reply:
We use the "best possible" version of the name and email taken from the committer or `Signed-off-by:` (full logic is in scripts/update_authors.pl). It is up to the individual contributor to decide what they wish to make public that way, and respect that.